Precomposition retracts of maps (#974)
UniMath/agda-unimath: src/orthogonal-factorization-systems/local-types.lagda.md

    map-distributive-Π-is-local-dependent-type (λ a _ → B a)
```

> ### If `S` is `g`-local, and `f` is a retract of `g` then `S` is `f`-local

Author: ```suggestion
### If `S` is `f`-local then `S` is local at every retract of `f`
```
